content/docs/reference/spec/migration/platform-api-0.5-0.6.md

```diff
@@ -0,0 +1,35 @@
++++
+title="Platform API 0.5 -> 0.6"
++++
+
+<!--more-->
+
+This guide is most relevant to platform operators.
+
+See the [spec release](https://github.com/buildpacks/spec/releases/tag/platform%2Fv0.6) for platform API 0.6 for the full list of changes and further details.
+
+## Platform Operator
+
+### Default process type not set if none is specified
+
+The `exporter` will no longer set the default process type if none is specified (even if there is only one process). Buildpacks implementing Buildpack API 0.6 or greater may set the default process type, or it may be specified by passing `-process-type` to the `exporter`.
+
+### Web is assumed to be default process type for older buildpacks
+
+As of Buildpack API 0.6, buildpacks may contribute the default process type by writing `default = true` in the process type definition in `<layers>/launch.toml`. To enable buildpacks implementing older Buildpack APIs to work with newer buildpacks, the lifecycle will assume that buildpacks that implement Buildpack API < 0.6 intended for `web` processes to be the default.
```

Comment:
It sounds like this item is simply additional information for the prior item "Default process type not set if none is specified", otherwise it looks weird to be speaking about a Buildpack API specific change.

Reply:
I can take this sentence out...

Reply:
Removed and combined with previous paragraph.